designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputNegativeInteger.kt

```diff
@@ -35,15 +39,20 @@ fun InputNegativeInteger(
     imeAction: ImeAction = ImeAction.Next,
     modifier: Modifier = Modifier,
 ) {
+    var inputValue by rememberSaveable { mutableStateOf(inputText ?: "") }
+    if (inputValue.startsWith('-')) inputValue = inputValue.removeRange(0, 1)
```

Comment:
This can be just replace by `inputValue.replaceFirst("-", "")`, WDYT?

Reply:
absolutely I wasn't aware of the existence of that method, please keep giving me these kind of tips 👍